.ci/hack/component_descriptor

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
```

Comment:
Are those hack scripts now copied and they have to be maintained in each repository using them or they are somehow synced from a single source repository?

Reply:
They are copied via `make tidy`. Essentially that happened in the past via `make revendor` and stored in the `vendor` folder. Unfortunately in the Councourse CI environment we can't rely on a golang environment to be present so we can't reference them dynamically via `go mod`/`go list`.

Reply:
That is very nice, thanks for the clarification!